osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -177,15 +174,14 @@ public override void OnEntering(IScreen last)
             base.OnEntering(last);
             buttons.FadeInFromZero(500);
 
-            Track = Beatmap.Value.Track;
             var metadata = Beatmap.Value.Metadata;
 
-            if (last is IntroScreen && Track != null)
+            if (last is IntroScreen && musicController.TrackLoaded)
```

Comment:
.

Reply:
`Track` doesn't exist any more. did you want it added back? i think it's kinda cleaner as it is.

Reply:
Let's keep as is...